apps/web-giddh/src/app/home/components/profit-loss/profit-loss.component.html

```diff
@@ -25,36 +25,34 @@
                 </div>
             </div>
             <div class="chart-div" google-chart chart="chartData" *ngIf="!requestInFlight">
-                <perfect-scrollbar [scrollIndicators]="true">
                     <chart [options]="profitLossChart" *ngIf="dataFound" #currentRatio></chart>
                     <div class="noDataAvailable" *ngIf="!dataFound">
                         <img src="{{imgPath}}noDataChart.svg" alt="no data found">
                     </div>
                     <div class="clearfix mrB2">
                         <div class="">
-                            <span class="mrB1 dueAmount">Net Profit/Loss</span>
-                            <span class="total_amt pull-right">
+                            <span class="mrB1 due-amount">Net Profit/Loss</span>
```

Comment:
use mrB1 to  mb-1

Reply:
CHeck please it's double green and i am worked on Dark green. Hope you understand